### PR TITLE
Integrate SHTC3 Sensor using TX/RX Commands. 

### PR DESCRIPTION
**Pull request by Naquino14**
Big W. Everything appears to be ok. 
I also did some code style changes and documented stuff the way @AarC10 wanted it.
For a quick summary of what happened:
SHTC3 Does not have registers and instead relies on sending a command for how you want to read data and immediately receiving it. This PR integrates the new TX/RX commands for use with the sensor. 
🚀

### CODE REVIEW

File: device/peripherals/SHTC3/SHTC3.h [hunk 1]
```diff
@@ -2,190 +2,284 @@
...
+     * @brief Toggle the sleep mode of the sensor
+     *
+     * @param setSleep Whether to set the sensor to sleep or not
+     * @return RetType The scheduler status
+     */
     RetType toggleSleep(bool setSleep) {
         RESUME();
 
-        RetType ret;
+        static RetType ret;
```
Comment (AarC10) [inline device/peripherals/SHTC3/SHTC3.h]: RetTypes dont need to be static

File: device/peripherals/SHTC3/SHTC3.h [hunk 2]
```diff
@@ -2,190 +2,284 @@
...
-        this->inLowPowerMode = lowPowerMode;
-    }
-
+    //    RetType setPowerMode(bool lowPowerMode) {
+    //        this->inLowPowerMode = lowPowerMode;
+    //    }
 
-private:
-    I2CDevice *mI2C;
+   private:
```
Comment (AarC10) [inline device/peripherals/SHTC3/SHTC3.h:255]: Fix this indent

File: device/peripherals/SHTC3/SHTC3.h [hunk 3]
```diff
@@ -2,190 +2,284 @@
...
+    /* The I2C object */
+    I2CDevice &mI2C;
+    /* The I2C address of the sensor */
     I2CAddr_t addr;
+    /* Is the sensor in low power mode */
     bool inLowPowerMode;
-
+    /* The ID of the sensor */
+    uint16_t id;
```
Comment (AarC10) [inline device/peripherals/SHTC3/SHTC3.h:263]: Maybe don't need the id as a field. You could probably just make this a static variable inside the init function

**Review by AarC10 — CHANGES_REQUESTED** [review]
Other than fixing your style should be good to go! Good job

File: device/peripherals/SHTC3/SHTC3.h (hunk 2 above)
Comment (Naquino14) [reply device/peripherals/SHTC3/SHTC3.h:255]: My auto formatter says it's correct

File: device/peripherals/SHTC3/SHTC3.h (hunk 3 above)
Comment (Naquino14) [reply device/peripherals/SHTC3/SHTC3.h:263]: It works right now. Im going to leave it as is.

File: device/peripherals/SHTC3/SHTC3.h (hunk 1 above)
Comment (Naquino14) [reply device/peripherals/SHTC3/SHTC3.h]: Fixed

**Review by AarC10 — APPROVED** [review]

**Pull request merged**